src/Common/tests/System/Net/Configuration.Certificates.cs

```diff
@@ -18,6 +18,8 @@ public static partial class Certificates
         {
             private const string CertificatePassword = "testcertificate";
             private const string TestDataFolder = "TestData";
+            private static Mutex m = new Mutex(false, "Global\\CoreFXTest.Configuration.Certificates.LoadPfxCertificate");
```

Comment:
Plus, to do it in the product, seems like we'd either a) need to serialize all imports across all processes, or b) serialize across all processes per path.  (a) would add a huge amount of unnecessary serialization, and (b) would require a potentially very large number of mutexes.  All to work around a bug, and with a workaround that end code could apply just as easily, as we're doing here in the test.